### PR TITLE
gathered all file reading/writing to one place

### PR DESCRIPTION
**Pull request by amazingalek**

### CODE REVIEW

File: OWML.ModHelper/ModStorage.cs [hunk 1]
```diff
@@ -1,52 +1,26 @@
...
-                return default;
-            }
-            try
-            {
-                var json = File.ReadAllText(path);
-                return JsonConvert.DeserializeObject<T>(json);
-            }
-            catch (Exception ex)
-            {
-                _console.WriteLine($"Error while loading {path}: {ex}");
```
Comment (amazingalek) [inline OWML.ModHelper/ModStorage.cs:33]: With this change, the error messages are lost

Comment (amazingalek) [reply OWML.ModHelper/ModStorage.cs:33]: I guess we'll still get them with verbose mode enabled (which should be the new default soon)

Comment (Raicuparta) [reply OWML.ModHelper/ModStorage.cs:33]: You can have a better error message in JSONHelper.cs though, no? Or would that be pointless with verbose mode?

Comment (amazingalek) [reply OWML.ModHelper/ModStorage.cs:33]: Don't have a console, and this is (currently) static.
Could use ModConsole.Instance (obsolete/deprecated) but even that isn't always available (before manifest is loaded).

Comment (amazingalek) [reply OWML.ModHelper/ModStorage.cs:33]: Not sure if verbose mode is enough, couldn't think of an easy way of testing it.

**Comment by amazingalek** [comment]
Will change base branch to dev after https://github.com/amazingalek/owml/pull/147 is merged

**Pull request merged**